elastic/kibana
# [Fleet] Migrate installPackageByUpload and installCustomPackage to state machine and clean up

Closes https://github.com/elastic/kibana/issues/189346
Closes https://github.com/elastic/kibana/issues/189347

## Summary
Final PR for part I of package lifecycle improvements. 

- Migrate both `installPackageByUpload` and `installCustomPackage` to `installPackageWithStateMachine` and clean up the old code. As the state machine is been working in production for two releases, I am confident in using it for the custom and upload packages without a feature flag.

### Testing

- Try installing a custom package
- Try installing a package via upload

They should work as usual

### Checklist
